mkdocs.yml

```diff
@@ -17,6 +17,7 @@ nav:
         - Positron Emission Tomography: modality-specific-files/positron-emission-tomography.md
         - Microscopy: modality-specific-files/microscopy.md
         - Near-Infrared Spectroscopy: modality-specific-files/near-infrared-spectroscopy.md
+        - Motion: modality-specific-files/motion.md
```

Comment:
should each page also be listed in the redirect map of the mkdocs.yml?

Reply:
It was like this, so I just added the motion. But happy to change it to what makes most sense :)

Reply:
@Remi-Gau would that make sense? After all, motion has never been introduced to the spec with something like `'04-modality-specific-files/0X-motion.md'` but gets into the spec with `modality-specific-files/motion.md`